openmetadata-docs/content/v1.4.x/deployment/backup-restore-metadata.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Backup Metadata
+slug: /deployment/backup-restore-metadata
+---
+
+# Backup & Restore Metadata
```

Comment:
We should have this file on 1.3.x directly since we're not bumping the support in the backup CLI

Reply:
also for new releases docs, we usually copy-paste the 1.3.x directory and create `1.4.x-SNAPSHOT`, which won't appear on the published site

Reply:
added 1.4.x docs in [separate PR](https://github.com/open-metadata/OpenMetadata/pull/15506)

Reply:
@pmbrull so just replace the existing file in 1.3?

Reply:
yes, since we know this is a better approach and our Python CLI is showing problems already

Reply:
we can just call this from now on